vms/platformvm/txs/mempool/mempool.go

```diff
@@ -272,7 +288,8 @@ func (m *mempool) PeekStakerTx() *txs.Tx {
 		return nil
 	}
 
-	return m.unissuedStakerTxs.Peek()
+	_, tx, _ := m.unissuedStakerTxs.Newest()
```

Comment:
Do we still need the above length check?

Reply:
Removed